pkg/ccl/streamingccl/streamingest/stream_ingestion_dist.go

```diff
@@ -807,15 +807,12 @@ func constructStreamIngestionPlanSpecs(
 			SrcInstanceID:     base.SQLInstanceID(partition.SrcInstanceID),
 			DestInstanceID:    destID,
 		}
-		streamIngestionSpecs[destID].PartitionSpecs[partition.ID] = partSpec
-		trackedSpans = append(trackedSpans, partition.Spans...)
-	}
-
-	// Remove any ingestion processors that haven't been assigned any work.
-	for key, spec := range streamIngestionSpecs {
```

Comment:
did you intentionally get rid of this loop which cleans up empty destination processors?

Reply:
Yes, there are no empty destination processors anymore, since we no longer blindly make one on every node to start.